android/src/main/java/io/sentry/react/RNSentryTimeToDisplay.java

```diff
@@ -0,0 +1,28 @@
+package io.sentry.react;
+
+import com.facebook.react.bridge.Promise;
+
+import android.view.Choreographer;
+
+import org.jetbrains.annotations.NotNull;
+import io.sentry.SentryDate;
+import io.sentry.SentryDateProvider;
+import io.sentry.android.core.SentryAndroidDateProvider;
+
+public class RNSentryTimeToDisplay {
+    public static void GetTimeToDisplay(Promise promise, SentryDateProvider dateProvider) {
+        try {
+            Choreographer choreographer = Choreographer.getInstance();
```

Comment:
We should run the `Choreographer.getInstance()` on the Main Thread as this function returns instance for the current thread.

https://github.com/getsentry/sentry-java/blob/b11dc55cfc01379e481be60de78b512191128a25/sentry-android-core/src/main/java/io/sentry/android/core/internal/util/SentryFrameMetricsCollector.java#L121
https://developer.android.com/reference/android/view/Choreographer#getInstance()